### PR TITLE
change title, delete feedback per node, pad login, favicon error is a…

### PR DESCRIPTION
**Pull request by rosasck**
… per browser error

Little bugs all around

### CODE REVIEW

**Comment by ericharper1** [comment]
What in here should I be testing?

**Comment by rosasck** [comment]
hmm to answer ur question.. You can look at the title in any tab and say 
- do we like the title? and the camel?
- when you log in do you enjoy the padding or is it insulting?
- and the favicon error is now gone bcuz we have a favicon!!!

**Comment by rosasck** [comment]
Also i only manually addded the favicon to the HTML pages that didnt extend the base

**Review by kaigoos — COMMENTED** [review]
Can we make the tab for student management just say "Students"?

File: simcon_project/templates/conversation/conversation_end.html [hunk 1]
```diff
@@ -7,8 +7,9 @@
         <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
         <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
         <title>
-            {% block title %}Simulated Conversations | Conversation End{% endblock %}
+            {% block title %}SimCon| Conversation{% endblock %}
```
Comment (devin-w-hill) [inline simcon_project/templates/conversation/conversation_end.html]: Missing space after SimCon

File: simcon_project/templates/conversation/conversation_step.html [hunk 2]
```diff
@@ -12,8 +12,9 @@
         <script src="{% static 'conversation_templates/disable_back_button.js' %}"></script>
         <script src="{% static 'js/jquery.bootstrap.modal.forms.js' %}"></script>
         <title>
-            {% block title %}Simulated Conversations | Conversation Step{% endblock %}
+            {% block title %}SimCon| Conversation{% endblock %}
```
Comment (devin-w-hill) [inline simcon_project/templates/conversation/conversation_step.html]: Missing space

File: simcon_project/templates/researcher_registration.html [hunk 3]
```diff
@@ -14,8 +14,9 @@
     </style>
 
     <title>
-    {% block title %}Simulated Conversations | Student{% endblock %}
+    {% block title %}SimCon| Registration{% endblock %}
```
Comment (devin-w-hill) [inline simcon_project/templates/researcher_registration.html]: Missing space

File: simcon_project/templates/student_settings_view.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
 
 
     <title>
-    {% block title %}Simulated Conversations | Settings{% endblock %}
+    {% block title %}SimCon| Settings{% endblock %}
```
Comment (devin-w-hill) [inline simcon_project/templates/student_settings_view.html]: Missing space

File: simcon_project/templates/view_response.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
         <title>
-            {% block title %}Simulated Conversations | Researcher{% endblock %}
+            {% block title %}SimCon| Response View{% endblock %}
```
Comment (devin-w-hill) [inline simcon_project/templates/view_response.html]: Missing space

File: simcon_project/templates/conversation/conversation_start.html [hunk 6]
```diff
@@ -6,9 +6,9 @@
         <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
         <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
         <title>
-            {% block title %}Simulated Conversations | Conversation Start{% endblock %}
+            {% block title %}SimCon | Conversation {% endblock %}
         </title>
-
+        <link rel="shortcut icon" href="{% static './title_image/favicon.ico' %}" type="image/x-icon">
```
Comment (devin-w-hill) [inline simcon_project/templates/conversation/conversation_start.html:12]: page is missing {% load static %}

File: simcon_project/templates/conversation/conversation_end.html [hunk 7]
```diff
@@ -7,8 +7,9 @@
...
         </title>
+    <link rel="shortcut icon" href="{% static './title_image/favicon.ico' %}" type="image/x-icon">
```
Comment (devin-w-hill) [inline simcon_project/templates/conversation/conversation_end.html:13]: missing {% load static %}

File: simcon_project/templates/feedback/view_feedback.html [hunk 8]
```diff
@@ -19,10 +19,9 @@
```
Comment (devin-w-hill) [inline simcon_project/templates/feedback/view_feedback.html:20]: {% block content %} is in a weird place in this file and i think it's keeping the right title from showing up. also don't think we need the bootstrap import here.

File: simcon_project/templates/student_settings_view.html [hunk 9]
```diff
@@ -19,7 +19,7 @@
...
     </title>
       <script src="{% static 'js/jquery.bootstrap.modal.forms.js' %}"></script>
```
Comment (devin-w-hill) [inline simcon_project/templates/student_settings_view.html:25]: The h1 tag here should be removed to be similar to researcher settings.

**Review by devin-w-hill — DISMISSED** [review]
lots of minor changes, looks great on the whole

File: simcon_project/templates/student_settings_view.html (hunk 9 above)
Comment (rosasck) [reply simcon_project/templates/student_settings_view.html:25]: ill delete it!

**Review by Likhovodov — APPROVED** [review]
Other than the things mentioned above, looks good

**Review by kaigoos — APPROVED** [review]

**Pull request merged**